media/js/products/vpn/invite.es6.js

```diff
@@ -52,6 +52,14 @@ const WaitListForm = {
                 newsletter: newsletter
             });
         }
+
```

Comment:
Note for reviewer: this file doesn't seem to be used anywhere (I think?), but I updated it anyway